### PR TITLE
September Doc Cleanup based on Github issues

### PR DESCRIPTION
**Pull request by lkasso**
### Context
Handles Github issues and a few internal issues in sections that have been recently updated in the Gradle User Manual.

This is a documentation change ONLY.

### Fixes:
- Continuous build description - https://github.com/gradle/gradle/issues/26252
- Document GRADLE_LIBS_REPO_OVERRIDE - https://github.com/gradle/gradle/issues/24217
- Deprecated alternative to `setOutputFile` - https://github.com/gradle/gradle/issues/25586
- Fix build order preference - https://github.com/gradle/gradle/issues/26088
- How to use system file watch with symlinks - https://github.com/gradle/gradle/issues/26201
- Update *Centralizing repositories declaration* section of [Dependency Management](https://docs.gradle.org/current/userguide/dependency_management.html) for @erichaagdev 
- Update admonition block CSS for @alllex 
- Fix javadocs CSS for search background color for @lkasso 

### Reviewers:
Under development.

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc [hunk 1]
```diff
@@ -490,22 +490,52 @@ As the way to declare the repositories and what they are expected to contain dep
...
 include::sample[dir="snippets/artifacts/defineRepositoryInSettings/kotlin",files="settings.gradle.kts[tags=declare_repositories_settings]"]
 include::sample[dir="snippets/artifacts/defineRepositoryInSettings/groovy",files="settings.gradle[tags=declare_repositories_settings]"]
 ====
 
-The `dependencyResolutionManagement` repositories block accepts the same notations as in a project, which includes Maven or Ivy repositories, with or without credentials, etc.
+The `dependencyResolutionManagement` repositories block accepts the same notations as in a project. This includes Maven or Ivy repositories, with or without credentials, etc.
 
 By default, repositories declared by a project will *override* whatever is declared in settings.
-You can change this behavior to make sure that you always use the settings repositories:
+This means that repositories declared in a project `build.gradle(.kts)` file *override* those declared in the `settings.gradle(.kts)` file:
```
Comment (erichaagdev) [inline subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc:508]: I think the second sentence is a little redundant and doesn't add much over the previous sentence aside from that it also mentions the build/settings file names. You could drop it or merge it.

File: subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc [hunk 2]
```diff
@@ -490,22 +490,52 @@ As the way to declare the repositories and what they are expected to contain dep
...
+There are three modes for dependency resolution management:
+
+[cols="1,1,1,1"]
+|===
+|Mode |Description |Default? |Use-Case
+
+|`PREFER_PROJECT`
+|Any repository declared on a project will cause the project to use the repositories declared by the project, ignoring those declared in settings.
+|Yes
+|Useful when teams need to use many different plugins not shared among subprojects.
```
Comment (erichaagdev) [inline subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc:525]: This doesn't have anything to do with plugins. Perhaps this is what you mean?:

```suggestion
|Useful when teams need to use different repositories not common among subprojects.
```

File: subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc [hunk 3]
```diff
@@ -490,22 +490,52 @@ As the way to declare the repositories and what they are expected to contain dep
...
+
+|`PREFER_SETTINGS`
+|Any repository declared directly in a project, either directly or via a plugin, will be ignored.
+|No
+|Useful for enforcing large teams to use the same plugin versions.
```
Comment (erichaagdev) [inline subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc:530]: I would expect the use case here to be the same as `FAIL_ON_PROJECT_REPOS` just that this option wouldn't fail the build.

```suggestion
|Useful for enforcing large teams to use approved repositories only, but will not fail the build when a project or plugin declares a repository.
```

File: subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc [hunk 4]
```diff
@@ -490,22 +490,52 @@ As the way to declare the repositories and what they are expected to contain dep
...
+
+|`FAIL_ON_PROJECT_REPOS`
+|Any repository declared directly in a project, either directly or via a plugin, will trigger a build error.
+|No
+|Useful for enforcing large teams to use approved and secure plugins only.
```
Comment (erichaagdev) [inline subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc:535]: I would drop the bit about "secured" as it doesn't really have anything to do with security.

```suggestion
|Useful for enforcing large teams to use approved repositories only.
```

File: subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc [hunk 5]
```diff
@@ -515,23 +545,16 @@ include::sample[dir="snippets/artifacts/defineRepositoryInSettings/groovy",files
 
 [[sub:fail_build_on_project_repositories]]
 
-If, for some reason, a project or a plugin declares a repository in a project, Gradle would warn you.
-You can however make it _fail the build_ if you want to enforce that only settings repositories are used:
+If, for some reason, a project or a plugin declares a repository in a project, Gradle will warn you.
```
Comment (erichaagdev) [inline subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc:548]: IMO `for some reason` doesn't add much value here.

```suggestion
If a project or a plugin declares a repository in a project, Gradle will warn you.
```

or

```suggestion
Gradle will warn you if a project or a plugin declares a repository in a project,.
```

**Review by erichaagdev — APPROVED** [review]
I only reviewed the changes to `declaring_repositories.adoc` and it LGTM. Nice job! 👍

**Comment by lkasso** [comment]
@bot-gradle test and merge

**Pull request merged**